components/sections/ClientLanding.tsx

```
@@ -0,0 +1,390 @@
+  const [openFaq, setOpenFaq] = useState<number | null>(null);
+  return (
+    <section className='w-full max-w-6xl mx-auto py-16 md:py-32 px-5 md:px-10 text-white'>
+      {/* Hero Section with Gradient and Split Layout */}
+      <div className='relative mb-20 flex flex-col md:flex-row items-center gap-10 md:gap-0'>
+        <div className='flex-1 fc gap-4 z-10'>
+          <motion.h1
+            initial={{ opacity: 0, y: 30 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ duration: 0.7 }}
+            className='text-4xl md:text-6xl font-bold text-left md:text-left bg-gradient-to-br from-violet-400 via-indigo-300 to-white bg-clip-text text-transparent'>
+            Let's Build Something Exceptional
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)